loops_lab.js

```diff
@@ -0,0 +1,325 @@
+
+
+
+console.log("New Version 7/2/2019");
+
+
+// # Loops Lab
+//
+// To complete this lab, do the following:
+//
+// 1. Fork this repo
+// 2. Clone **your** forked repo
+// 3. Create a file called `loops_lab.js` inside your cloned repo
+// 4. Copy the description of the exercise as a comment
+// 5. Underneath the commented description write your code.
+// 6. Ensure your solutions work properly!
+// 7. Open PR against this repo.
+//
+// ___
+//
+// ## Exercises
```

Comment:
Good demonstration on how to use for and while loops for differing problems. I don't have any major feedback other than some while loop statement having extra statement that are unnecessary. Keep up the good work!